include/openmapper/settings.h

```diff
@@ -0,0 +1,19 @@
+// Auto-generated headers file, do not edit, will be overwritten by cmake. 
```

Comment:
This file is auto generated, eventually is a good idea to add it to gitignore? eventually put it into the root folder as it will be created everytime